src/hotspot/share/utilities/elfFuncDescTable.cpp

```diff
@@ -46,7 +46,7 @@ ElfFuncDescTable::ElfFuncDescTable(FILE* file, Elf_Shdr shdr, int index) :
 ElfFuncDescTable::~ElfFuncDescTable() {
 }
 
-address ElfFuncDescTable::lookup(Elf_Word index) {
+address ElfFuncDescTable::lookup(Elf64_Addr index) {
```

Comment:
I think this could be `Elf_Addr` to be platform-specific. `Elf_Word` is either `Elf32_Word` or `Elf64_Word`, depending on the platform. But both are `typedefs` for `uint32_t` for some reason. That's why it must have complained here.